packages/email-components/src/components/RatingItem/README.md

```diff
@@ -0,0 +1,23 @@
+# Rating
+
+The `mjc-rating-item` is a component with the text below the icon and with the link. It uses for example setting the status of jobs offered in the e-mail template.
+
+```mjml
+<mjc-rating-item
+  alt="Smiley in love"
+  href="https://www.jobs.cz"
+  src="https://ijobs.cz/static/images/email/icons/smiley-in-love.png?1"
+  title="I'm hired"
+/>
+```
+
+## Available props for mjc-rating-item
+
+| Prop name     | Type      | Default | Required | Description                                                     |
+| ------------- | --------- | ------- | -------- | --------------------------------------------------------------- |
+| `alt`         | `string`  | -       | no       | Image alt description                                           |
+| `href`        | `string`  | -       | yes      | Link to url                                                     |
+| `isTableCell` | `boolean` | `false` | no       | This prop is for using this component with Rating component     |
+| `rel`         | `string`  | -       | no       | Relationship between a linked resource and the current document |
+| `src`         | `string`  | -       | yes      | Src to image                                                    |
```

Comment:
We try to avoid shorthands...

```suggestion
| `src`         | `string`  | -       | yes      | Image source                                                    |
```